### DISCUSSION TITLE
各位，我做了个纪录片，来看看吧

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#2502] [其他]
<img width="240" alt="截屏2024-08-17 12 13 05" src="https://github.com/user-attachments/assets/8ba8acfd-665e-4648-81d1-d5b50c8418cc">

【大型纪录片之小米&米哈游传奇-哔哩哔哩】 https://b23.tv/mIjMzt5

**Comment by Dave-12138** [comment]
markdown语法提示：使用`[显示文本](链接地址)`来发送链接<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >

**Comment by cyq20091029** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/03/28/f9ee6cc8f2d5c553d67992bac8c8417b_5627889348984592351.png" alt="砂金 赢" width="75" >
感谢大佬教学，现在可以跳转了
